packages/core/src/artifacts.ts

```diff
@@ -146,22 +146,6 @@ export async function generateCommittedArtifacts(
   return artifacts;
 }
 
-const nodeAPIJS = (
-  cwd: string,
-  config: KeystoneConfig
-) => `import keystoneConfig from '../../keystone';
-import { PrismaClient } from '.prisma/client';
-import { createQueryAPI } from '@keystone-6/core/___internal-do-not-use-will-break-in-patch/node-api';
```

Comment:
Are we going to remove `packages/core/src/___internal-do-not-use-will-break-in-patch/node-api.ts` in this PR?